src/main.cpp

```diff
@@ -36,6 +38,7 @@ void setup() {
   mdls->modules.push_back(sys);
   mdls->modules.push_back(pin);
   mdls->modules.push_back(lds);
+  mdls->modules.push_back(e131);
```

Comment:
Would be nicer if the modules could self-register rather than needing to edit main.cpp - not sure if C++ has anything easy we can use here without a big library

Reply:
Agree! It's already clumsy that the variable is defined in each SysMod*.h:
<img width="209" alt="image" src="https://github.com/ewoudwijma/LedFix/assets/1737159/1c6ca210-c70f-4174-985f-52de45e2af11">
And then created in main, don't know how to do better yet.
I also don't know why static is needed to be added here, otherwise linker errors...